DrillProcess.py

```diff
@@ -23,215 +19,143 @@
 ROTATION_SPEED = 2e3                # at 12 volts
 SAMPLE_HOLDER_HEIGHT = 100
 STOP_ABOVE_GROUND = 1 # mm
+DEADZONE = 0.2 #analog joysticks arent perfect
+TRIG_DEADZONE = 0.05
 
-### INITIALIZTION ###
+done = False
+
+
+'''### INITIALIZTION ###
 drill.storage.top_motor_movement = 0
 drill.storage.bottom_motor_movement = 0
 drill.storage.rotating = False
 drill.storage.top_distance = 0
 drill.storage.bottom_distance = 0
-drill.storage.carousel_position = 'EMPTY'
-
-### FUNCTIONS ###
-async def top_is_moving():
-    if drill.storage.top_motor_movement == 1:
-        return 'RAISING'
-    elif drill.storage.top_motor_movement == 2:
-        return 'LOWERING'
+drill.storage.carousel_position = 'EMPTY'''
+
+### CONSTANTS ###
+DRILL_RAISE_DUTY_CYCLE = 0.2*1e5    # at 12 volts
+DRILL_LOWER_DUTY_CYCLE = 0.1*1e5    # at 12 volts
+ROTATION_SPEED = 2e3                # at 12 volts
+SLEEP_DURATION = 0.1 #seconds
+controller_num = config.science_controller
```

Comment:
You'll have to add the `science_controller` variable to the `config` module in order to use it.